packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -280,43 +280,50 @@ function buildUrl(server: ServerObject): string {
 }
 
 /**
- * Returns object | array where all object keys are sanitized. Keys passed in
- * exceptions are not sanitized.
+ * Returns object/array/scalar where all object keys (if applicable) are
+ * sanitized.
  */
-export function sanitizeObjKeys(
-  obj: object | Array<any>,
-  exceptions: string[] = []
```

Comment:
This `exceptions` parameter is never used except for in one unit test. I don't see how we will ever support it so I will remove it for now.